src/parser/jobs/brd/modules/RefulgentProcs.tsx

```diff
@@ -0,0 +1,52 @@
+import {Trans, Plural} from '@lingui/react'
+import {ActionLink, StatusLink} from 'components/ui/DbLink'
+import {Procs} from 'parser/core/modules/Procs'
+import {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+export default class RefulgentProcs extends Procs {
+	trackedProcs = [
+		{
+			procStatus: this.data.statuses.STRAIGHT_SHOT_READY,
+			consumeActions: [this.data.actions.REFULGENT_ARROW],
+		},
+	]
+
+	protected addJobSpecificSuggestions() {
```

Comment:
Since you're not doing any job-specific logic for the suggestions (and since you're only tracking one proc status), you can actually get away with using the default dropped/overwritten proc suggestions and simply overriding the default property values. DNC has a good example of this that you can follow for the icon/content, and you can define your own overrides for the override[Dropped/Overwrote]ProcsWhy functions to take care of the why pieces.